Detectors/TPC/reconstruction/include/TPCReconstruction/HwClusterer.h

```diff
@@ -163,29 +185,25 @@ class HwClusterer : public Clusterer
   /// \param value  some value
   Vc::uint_v getFpOfADC(const Vc::uint_v value);
 
-  /// Does the comparison between two pads and sets the bits accordingly
-  /// \param qMaxIndex      Buffer index of central pad
-  /// \param compareIndex   Buffer index of pad to compare with
-  /// \param bitMax         Bit to be set for peak finding
-  /// \param bitMin         Bit to be set for minimum finding
-  /// \param row            Row number
-  void compareForPeak(const unsigned qMaxIndex, const unsigned compareIndex, const unsigned bitMax, const unsigned bitMin, const unsigned short row);
-
   /*
    * class members
    */
-  static const int mTimebinsInBuffer = 5;
+  static const int mTimebinsInBuffer = 6;
```

Comment:
This change went along with the change in line 213 (and 815) `computeClusterForTime(i - 2);` -> `computeClusterForTime(i` - 3);`

I've seen, that to calculate all the relationships of all cluster pads in advance, I need the 6th timebins and have look for clusters in 1 timebin later than before (-3 instead of -2). This is only now important because possible minima of the newest (last) timebin are taken into account for the splitting. Before, this information was simply not used and therefore it did not matter that it was not yet calculated (and therefore I did not see it before).

<img width="330" alt="bildschirmfoto 2018-10-19 um 20 02 21" src="https://user-images.githubusercontent.com/15690984/47235630-ea799280-d3d9-11e8-8295-ca3da873dc28.png">